### PR TITLE
replace putfile() function by put() & modify related tests

### PR DESCRIPTION
**Pull request by gallexis**
still need to fix one test

### CODE REVIEW

**Comment by gallexis** [comment]
fixes #92

File: exchange/tx_test.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package exchange
 import (
 	"context"
 	"fmt"
+	cidlink "github.com/ipld/go-ipld-prime/linking/cid"
```
Comment (tchardin) [inline exchange/tx_test.go:6]: This is a nit but standard lib is usually at the top and then external libs at the bottom

File: internal/testutil/mocknet.go [hunk 2]
```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	mh "github.com/multiformats/go-multihash"
```
Comment (tchardin) [inline internal/testutil/mocknet.go:7]: ^

**Review by tchardin — APPROVED** [review]
Looks good!

**Pull request merged**